### PR TITLE
Add Select samples

### PR DESCRIPTION
**Pull request by RivaIvanova**
Closes https://github.com/IgniteUI/igniteui-xplat-docs/issues/1058

### CODE REVIEW

File: samples/inputs/select/header/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <IgrSelectItem value="implement"><span>Implementation</span></IgrSelectItem>
+        <IgrSelectItem value="test" disabled><span>Testing</span></IgrSelectItem>
+        <IgrSelectItem value="docs"><span>Documentation</span></IgrSelectItem>
+      </IgrSelect>
+    </div>
+  );
+}
+
+// rendering above class to the React DOM
+const root = ReactDOM.createRoot(document.getElementById("root"));
```
Comment (tishko0) [inline samples/inputs/select/header/src/index.tsx:32]: Please change to single q's.

File: samples/inputs/select/overview/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <IgrSelectItem value="Apple"><span>Apple</span></IgrSelectItem>
+        <IgrSelectItem value="Banana"><span>Banana</span></IgrSelectItem>
+        <IgrSelectItem value="Mango"><span>Mango</span></IgrSelectItem>
+      </IgrSelect>
+    </div>
+  );
+}
+
+// rendering above class to the React DOM
+const root = ReactDOM.createRoot(document.getElementById("root"));
```
Comment (tishko0) [inline samples/inputs/select/overview/src/index.tsx:23]: Please update to single q's

File: samples/inputs/select/styling/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            <span slot="suffix">MX</span>
+          </IgrSelectItem>
+        </IgrSelectGroup>
+      </IgrSelect>
+    </div>
+  );
+}
+
+// rendering above class to the React DOM
+const root = ReactDOM.createRoot(document.getElementById("root"));
```
Comment (tishko0) [inline samples/inputs/select/styling/src/index.tsx:96]: Same as above. Single q's

File: samples/inputs/select/group/src/index.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            <span slot="suffix">MX</span>
+          </IgrSelectItem>
+        </IgrSelectGroup>
+      </IgrSelect>
+    </div>
+  );
+}
+
+// rendering above class to the React DOM
+const root = ReactDOM.createRoot(document.getElementById("root"));
```
[the remark this answers is not in the document]
Comment (skrustev) [reply samples/inputs/select/group/src/index.tsx:123]: Here's the fix regarding the quotation marks. If it is merged before this pr there won't be any changes required here:
https://github.com/IgniteUI/igniteui-react-examples/pull/450

**Review by tishko0 — APPROVED** [review]

**Pull request merged**